packages/client-core/src/user/components/MagicLink/AuthMagicLink.tsx

```diff
@@ -55,25 +59,73 @@ const AuthMagicLink = ({ token, type, instanceId, path }: Props): JSX.Element =>
     }
   }, [])
 
+  function doRedirect(connect = false) {
+    let redirect = /https:\/\//.test(path) ? path : config.client.clientUrl + path
+    console.log('added path', redirect)
+    if (instanceId != null) redirect += `?instanceId=${instanceId}`
```

Comment:
we have many more parameters than just `instanceId` (invites, roomcodes, studio project and scene path etc), do we want to make this generic somehow?

Reply:
As discussed, this component is just used for resolving magiclink logins, which currently can only redirect to instances. If we enable redirects to e.g. a specific project/scene in the studio, we'll deal with it at that time.